web_hosting_manager/app/components/app_logs.js

```diff
@@ -14,27 +14,27 @@ export default class AppLogs extends Component {
     };
   }
   componentDidMount() {
-    this.setState({loading: true});
-    this.props.getLogFilePath()
-      .then(path => {
+    this.setState({ loading: true }); // eslint-disable-line react/no-did-mount-set-state
+    this.props.getLogFilePath() // eslint-disable-line react/prop-types
+      .then(path => { // eslint-disable-line promise/always-return
```

Comment:
here we could simply `return` inside the promise to keep to linter guides.

Reply:
updated it josh..